support-frontend/assets/pages/digital-subscriber-thank-you/components/thankYouHeader.tsx

```diff
@@ -1,12 +1,21 @@
+import { css } from '@emotion/react';
+import { from, titlepiece } from '@guardian/source-foundations';
 import DirectDebitMessage from 'pages/supporter-plus-thank-you/components/thankYouHeader/directDebitMessage';
 import {
 	header,
 	headerSupportingText,
-	headerTitleText,
 } from 'pages/supporter-plus-thank-you/components/thankYouHeader/thankYouHeader';
 import Heading from './heading';
 import Subheading from './subheading';
 
+const headerTitleText = css`
+	${titlepiece.small()};
+	font-size: 24px;
+	${from.tablet} {
+		font-size: 40px;
+	}
+`;
+
```

Comment:
This was coupled to the `supporter-plus-thank-you` heading - if we want to maintain that, we should potentially create a new component.